services.html

```diff
@@ -0,0 +1,114 @@
+  <title>Rmedia</title>
+
+  <!-- Favicons -->
+  <link rel="apple-touch-icon" sizes="180x180" href="/resources/favicons/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/resources/favicons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/resources/favicons/favicon-16x16.png">
+  <link rel="manifest" href="/resources/favicons/site.webmanifest">
+  <link rel="mask-icon" href="/resources/favicons/safari-pinned-tab.svg" color="#5bbad5">
+  <link rel="shortcut icon" href="/resources/favicons/favicon.ico">
+  <meta name="msapplication-TileColor" content="#da532c">
+  <meta name="msapplication-config" content="/resources/favicons/browserconfig.xml">
+  <meta name="theme-color" content="#ffffff">
+</head>
+<body>
+  
+  <header>
+    <div class="container flex-box head-box">
+      <h1><a href="#"><img src="resources/images/logo.jpg" alt="Rmedia"></a></h1>
+      <div class="search-and-text">
+        <input type="search" name="search" id="search" placeholder="search...">
+        <div class="user-text-box flex-box">
+          <p class="welcome-text"></p>
+          <button class="logout styled-btn-white">Logout</button>
+        </div>
+      </div>
+    </div>
+  </header>
+
+  <section class="content">
+
+    <div class="container">
+
+      <div class="content-section flex-box">
+        <section class="left-section">
+          <div class="drop-down">
+            <button class="drop-down-button">Menu</button>
+          </div>
+          <nav class="navigation">
+            <ul id="nav-menus">
+             
+            </ul>
+          </nav>
+        </section>
+        <section class="middle-section">
+          <div class="top-section flex-box">
+            <div class="content-area">
+              <h2 id="page-heading"></h2>
+              <p id="blog-paragraph"> Lorem lpsum is simply dummy text of the printing and typesetting industry.
+              Lorem lpsum has been the industry's standard dummy text ever since the 1500's,when an
+              unknown printer took a galley of type and scrambled it to make a type specimen book.It has
+              survived not only five centuries, but also the leap into electronic typesetting,remaining
+              1960s with the release ofLetraset sheets containing lorem lpsum passages,and more recently with desktop publishing
+              software
+              like Aldus Pagemker including versions of lorem lpsum.</p>
+            </div>
+            <div class="image-area">
+              <img src="resources/images/main-image.jpg" alt="Rmedia image" class="content-img">
+            </div>
+          </div>
+          <div class="bottom-section">
+            <table>
+              <thead id="table-head">
+               
+              </thead>
+              
+              <tbody id="table-body">
+                
+              </tbody>
+              
+            </table>
```

Comment:
You can make this table element itself dynamic, by inserting it only if 'table' column configurations are available in service.json

Reply:
Entire table was not made dynamic because of the inability to bind an event listener to table head.